pkg/spdk/log.go

```diff
@@ -0,0 +1,145 @@
+package spdk
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	spdkclient "github.com/longhorn/go-spdk-helper/pkg/spdk/client"
+)
+
+const (
+	ServiceSpdkTgt = "spdk_tgt"
+)
+
+func svcLogSetLevel(spdkClient *spdkclient.Client, service, level string) error {
+	log := logrus.WithFields(logrus.Fields{
+		"service": service,
```

Comment:
Is `service` is just for info purpose? will we have any other chances of having another service other than spdk_tgt?

Reply:
That's OK. If you think it's helpful, let's keep it.